src/main/java/seedu/address/model/student/Student.java

```diff
@@ -0,0 +1,57 @@
+//@@author m-aslam-mj2
+package seedu.address.model.student;
+
+import java.util.Set;
+
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.tag.Tag;
+
+
+
+
+/**
+ * Represents a student in the LMS.
+ *
+ */
+
+public class Student extends Person {
+    /**
+     * Every field must be present and not null.
+     *
+     * @param name
+     * @param phone
+     * @param email
+     * @param address
+     * @param tags
+     */
+
+    private String courseCode;
+    private String matricNo;
+
+    public Student(Name name, Phone phone, Email email, Address address,
+                   Set<Tag> tags, String courseCode, String matricNo) {
```

Comment:
We will need to verify with Steph if this indentation is acceptable. When I asked Akshay about it, he told me to follow the 8 spacing rule for word-wrapping strictly as the code style is graded by a bot.